src/site/header.rs

```diff
@@ -36,14 +41,15 @@ fn nav(pages: &[String], path_prefix: &Option<String>) -> Result<Box<nav<String>
             .unwrap_or(file_path.as_os_str())
             .to_string_lossy();
 
-        let href = if let Some(prefix) = &path_prefix {
-            format!("/{}/{}.html", prefix, file_name)
-        } else {
-            format!("/{}.html", file_name)
-        };
+        let href = generate_prefix_link(path_prefix, file_name.to_string());
 
         html.extend(html!(<li><a href=href>{text!(file_name)}</a></li>));
     }
+
+    if let Some(Artifacts { cargo_dist: true }) = artifacts {
+        let href = generate_prefix_link(path_prefix, String::from("artifacts"));
+        html.extend(html!(<li><a href=href>{text!("Downloads")}</a></li>));
```

Comment:
```suggestion
        html.extend(html!(<li><a href=href>{text!("Install")}</a></li>));
```
? curious your thoughts